Shared/Samples/Apply symbology to shapefile/README.md

```diff
@@ -0,0 +1,46 @@
+# Apply symbology to shapefile
+
+Display a shapefile with custom symbology.
+
+![Image of apply symbology to shapefile](apply-symbology-to-shapefile.png)
+
+## Use case
+
+Feature layers created from shapefiles do not possess any rendering information, and will be assigned with a default symbology. You can apply custom styling to ensure that the content is visible and usable in the context of a specific map. For example, you could use this to visually differentiate between features originating from two different shapefiles, by applying a blue color to one, and a red color to the other.
+
+## How to use the sample
+
+Click the button to apply a new symbology renderer to the feature layer created from the shapefile.
```

Comment:
This line does not match how you have written the sample.

Reply:
The readme doesn't match [the .NET implementation](https://devtopia.esri.com/runtime/common-samples/blob/main/designs/ApplySymbologyToShapefile/implementation-details.md), which from what I can tell just displays the data with the custom symbology right away. Do you recommend that I just update our README here?

Reply:
Yeah, I'd say make our README match our sample.